tests/nlu/base/test_extractors.py

```diff
@@ -145,6 +145,72 @@ def test_crf_json_from_non_BILOU(spacy_nlp, ner_crf_pos_feature_config):
     assert rs[4] == {"start": 29, "end": 31, "value": "by", "entity": "where"}
 
 
+def test_crf_create_entity_dict(spacy_nlp):
+    from rasa.nlu.extractors.crf_entity_extractor import CRFEntityExtractor
+    from rasa.nlu.tokenizers.spacy_tokenizer import SpacyTokenizer
+
+    crf_extractor = CRFEntityExtractor()
+    tokenizer = SpacyTokenizer()
```

Comment:
Can you also add a `WhitespaceTokenizer` and assert that the result is the same for both?

Reply:
fixed here https://github.com/RasaHQ/rasa/pull/4018/commits/7427cc1935c2cb9806f27651f27a77a00ffa9747